src/app/core/store/collections/collections.effects.ts

```diff
@@ -295,6 +297,53 @@ export class CollectionsEffects {
     })
   );
 
+  @Effect()
+  public createAttributes$: Observable<Action> = this.actions$.pipe(
+    ofType<CollectionsAction.CreateAttributes>(CollectionsActionType.CREATE_ATTRIBUTES),
+    mergeMap(action => {
+      const attributesDto = action.payload.attributes.map(attr => CollectionConverter.toAttributeDto(attr));
+      const correlationIdMap = action.payload.attributes.reduce((acc, attr) => {
+        acc[attr.name] = attr.correlationId;
+        return acc;
```

Comment:
Why do we need correlation IDs if attributes are still matched based on their names?